clang/test/APINotes/Inputs/Frameworks/CXXInteropKit.framework/Headers/CXXInteropKit.apinotes

```diff
@@ -12,4 +12,21 @@ Enumerators:
   SwiftName: red
 Tags:
 - Name: NSSomeEnumOptions
-  SwiftName: SomeEnum.Options
\ No newline at end of file
+  SwiftName: SomeEnum.Options
+- Name: GlobalStruct
+  Methods:
+  - Name: 'method'
+    SwiftName: globalMethod()
+    ImportAs: unsafe
+- Name: ParentNS.ChildNS.ParentStruct.ChildStruct
+  Methods:
+    - Name: 'method'
+      SwiftName: nestedMethod()
+- Name: ImportAsReference
+  ImportAs: reference
```

Comment:
The naming convention for the attributes you've chosen is fine, but it doesn't cover the fact that these attributes are Swift-specific. We should not pollute the general API notes attribute namespace with Swift-specific attributes as the general practice has been using the clang-attribute like names instead (e.g. Availability API notes annotation applies to availability attribute, and `SwiftName` applies to swift_name attribute respectively).
I would suggest renaming the new attributes to be Swift-specific, for example, `SwiftImportAs`, `SwiftReferenceRetain` and `SwiftReferenceRelease`.